temp.py

```diff
@@ -3,8 +3,7 @@
 import tensorflow as tf
 
 ssl._create_default_https_context = ssl._create_unverified_context
-token = 'https://discord.com/api/webhooks/771253998313144321/5FNBsAi8-exyc-3rx' \
-        'LOYudQdQqMdpcaCOqCh6z1IX4ledG1oGyaONeIY1HDUns6qBZhW'
+token = 'https://discord.com/api/webhooks/885143691201773578/dJgoeeuYqhhCoBNC5nX5ZMBAlx05hKkXVAgSWrSUk-rPc-c7P4n0DL98yGdHwHbERnj4'
```

Comment:
Please replace it with token = {DISCORD_TOKEN}

Reply:
like with which token 🤔.

Reply:
Just replace line 6 with below
`token = {DISCORD_TOKEN}`

Reply:
I am asking what is DISCORD_TOKEN and form where I can find it?

Reply:
It's a webhook from discord and I don't wanna expose my tokens anymore so I'm storing the token in the Github secrets